shared/js/background/before-request.es6.js

```diff
@@ -97,6 +97,17 @@ function handleRequest (requestData) {
     // don't have a tab instance for this tabId or this is a new requestId.
     if (requestData.type === 'main_frame') {
         if (!thisTab || thisTab.requestId !== requestData.requestId) {
+            if(thisTab) {
```

Comment:
I'm suprised ESLint didn't catch this, perhaps the GitHub action isn't working?

Reply:
👍  Went through and cleaned up some lint errors in general